components/_buttons/buttons.scss

```diff
@@ -96,6 +96,34 @@
       box-shadow: 0 0 0 0.25rem $rgba-3;
     }
   }
+
+  &--secondary-dark {
+    color: $white;
+    border-color: $white;
+    bottom: 0;
+    transition: background 0.2s, bottom 0.2s;
```

Comment:
Double check these actually do something

Reply:
transition: background 0.2s, bottom 0.2s;

Reply:
apparently its needed, I couldn't see any changes without it, but I think it might be some small detail im missing so I would rather leave the transition